Makefile

```diff
@@ -46,6 +57,12 @@ starknet-test: starknet-clean
 starknet-deploy: starknet-build
 	@. ./contracts/starknet/.env && . ./contracts/starknet/deploy.sh
 
+starknet-deploy-and-connect: starknet-build
```

Comment:
Propagated to where?

Note the command is a oneliner connected by "&&".
It was implemented this way because the other way (using .ONESHELL and executing one line at a time) was having unpredictable behaviour with bugs I could not fix.

This way is still clean, tidy, and has more atomic and robust behaviour.

Reply:
refactored make targets to new "&& \" format, instead of ONESHELL as new format has been more reliable